public/img/icons/README.md

```diff
@@ -0,0 +1,8 @@
+# How to add a new icon
+
+- Add the new icon svg to the `unicons/` directory
```

Comment:
this is almost certainly wrong, but what should we put here?

essentially we have to explain the logic behind: https://github.com/grafana/grafana/blob/ash/81142/packages/grafana-ui/src/components/Icon/utils.ts#L13

`Icon` defaults to the `unicons` directly hence why most icons get put in here, but we probably need some logical definition of what icons fit where.

wdyt @grafana/grafana-frontend-platform ?

Reply:
Wasn't it so if the icon is the one from [here](https://iconscout.com/unicons/solid-icons), it'd be added to the `unicons` folder. If it's a custom made icon, then it'd be in the `custom` folder? Also we'd probably get rid of the `solid` and `mono` folders and just use the unicons one.

Reply:
I agree with @Clarity-89.